WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteCredentialsViewController.swift

```diff
@@ -24,6 +24,9 @@ class SiteCredentialsViewController: LoginViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        navigationItem.title = WordPressAuthenticator.shared.displayStrings.logInTitle
+        styleNavigationBar(forUnified: true)
+
 		localizePrimaryButton()
```

Comment:
is it just my browser or GitHub is messing up with the indentation? When I open it in Xcode, everything is in order anyway.

Reply:
I noticed that too when Thuy first added this file. it appeared all kinds of wrong on the PR, but in code it's fine. 🤷 .